feeds/ipq807x/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -84,11 +84,17 @@ emmc_do_upgrade_bootconfig() {
 	fi
 
 	for part in "0:BOOTCONFIG" "0:BOOTCONFIG1"; do
-		local emmcblock=$(find_mmc_part $part)
-		echo erase ${emmcblock}
-		dd if=/dev/zero of=${emmcblock} 2> /dev/null
-		echo update $emmcblock
-		dd if=/tmp/bootconfig of=${emmcblock} 2> /dev/null
+               local emmcblock=$(echo $(find_mtd_chardev $part) | sed 's/^.\{5\}//')
```

Comment:
emmcblock -> mtdchar ?

Reply:
> emmcblock -> mtdchar ?

just use same name for mtd cha and emmcblock.